src/ui/templates/components/address-input.njk

```diff
@@ -0,0 +1,128 @@
+{% from "govuk/components/input/macro.njk" import govukInput %}
+
+{% macro render(params) %}
+
+  {% set ADDRESS_LINE_ONE_FIELD = params.addressLineOne %}
+  {% set ADDRESS_LINE_TWO_FIELD = params.addressLineTwo %}
+  {% set COUNTY_FIELD = params.country %}
+  {% set TOWN_FIELD = params.town %}
+  {% set POSTCODE_FIELD = params.postcode %}
+  {% set validationErrors = params.validationErrors %}
+  {% set application = params.application %}
+  {% set submittedValues = params.submittedValues %}
+
```

Comment:
Could we call this component `address-fields`?

Then i'm thinking we could be more DRY with each field, they all look the same with just a different ID?f

Could have a new component/macro `address-field-input` which renders a `govukInput` like below, but passing an ID, then below could just have e.g

```js
{{ addressFieldInput.render({
  id: ADDRESS_LINE_ONE_FIELD
}) }}
```